ChoseYourOwnAdventure/Page.java

```diff
@@ -0,0 +1,40 @@
+package ChoseYourOwnAdventure;
+
+import java.util.ArrayList;
+
+public class Page {
+    private String text;
+    private Page p1;

```

Comment:
only two pages availble ? what if there are 3 choices ?
whem you number your var names i allways know you made something not very generic